stdlib/public/core/Array.swift

```diff
@@ -429,24 +429,25 @@ extension Array: RandomAccessCollection, MutableCollection {
   ///     print(numbers[i])
   ///     // Prints "50"
   ///
-  /// The value passed as `n` must not offset `i` beyond the bounds of the
-  /// collection.
+  /// The value passed as `distance` must not offset `i` beyond the bounds of
+  /// the collection.
```

Comment:
a. Shouldn't this be listed as a precondition?
b. The phrasing is a bit weird, since the value passed doesn't itself offset anything.  Can we do better?

Reply:
Agreed! I'm going to take this up on a different branch, since that's existing language and this branch keeps falling behind master.